### PR TITLE
AArch64: Print FMOV (imm) instructions in ARM64Debug.cpp

### PR DESCRIPTION
**Pull request by knn-k**
This commit adds support for printing FMOV (imm) instructions in
ARM64Debug.cpp.  The instructions use a special 8-bit format for
representing floating point values.

Signed-off-by: KONNO Kazuhiro <konno@jp.ibm.com>

### CODE REVIEW

**Comment by knn-k** [comment]
@genie-omr build aarch64

File: compiler/aarch64/codegen/ARM64Debug.cpp [hunk 1]
```diff
@@ -860,6 +880,11 @@ TR_Debug::print(TR::FILE *pOutFile, TR::ARM64Trg1ImmInstruction *instr)
          }
       trfprintf(pOutFile, ", " POINTER_PRINTF_FORMAT, (instr->getBinaryEncoding() + offset));
       }
+   else if ((op == TR::InstOpCode::fmovimms) || (op == TR::InstOpCode::fmovimmd))
+      {
+      uint32_t imm = instr->getSourceImmediate() & 0xFF;
+      trfprintf(pOutFile, ", %lf", getDoubleFromImm8(imm));
```
Comment (0xdaryl) [inline compiler/aarch64/codegen/ARM64Debug.cpp:886]: Is it useful to print the immediate in addition to the conversion to double?  Yes, the conversion is convenient, but sometimes you may want to know exactly what value was encoded in the instruction.

Comment (knn-k) [reply compiler/aarch64/codegen/ARM64Debug.cpp]: I updated the code and it now prints like this:
```
fmovimmd s1, 0x60 (0.500000)
```

**Comment by knn-k** [comment]
@genie-omr build aarch64

**Pull request merged**